docs/design/lexical_conventions/string_literals.md

```diff
@@ -292,30 +298,11 @@ var String: z = ##"Raw strings #"nesting"#"##;
 var String: w = #"Tab is expressed as \t. Example: '\#t'"#;
 ```
 
-Note that both a raw simple string literal and a raw block string literal can
-begin with `#"""`. These cases can be distinguished by the presence or absence
-of additional `"`s later in the same line:
-
--   In a raw simple string literal, there must be a `"` and one or more `#`s
-    later in the same line terminating the string.
--   In a raw block string literal, the rest of the line is a file type
-    indicator, which can contain neither `"` nor `#`.
-
-```carbon
-// This string is a single-line raw string literal.
-// The contents of this string start and end with exactly two "s.
-var String: ambig1 = #"""This is a raw string literal starting with """#;
-
-// This string is a raw block string literal with file-type 'This', whose
-// contents start with "is a ".
-var String: ambig2 = #"""This
-  is a block string literal with file type 'This', first character 'i',
-  and last character 'X': X\#
-  """#;
-
-// This is a single-line raw string literal, equivalent to "\"".
-var String: ambig3 = #"""#;
-```
+Extending simple block string literals, a valid raw block string literal opening
+delimiter (for example, #""") always results in block string handling. For
+example, #"""# begins a raw block string literal with a file type indicator of
+#, rather than the possible interpretation as a raw simple string literal. This
+is true even when the file type indicator is invalid, as with #"""foo"""#.
```

Comment:
```suggestion
Extending simple block string literals, a valid raw block string literal opening
delimiter (for example, `#"""`) always results in block string handling. For
example, `#"""#` begins a raw block string literal with a file type indicator of
`#`, rather than the possible interpretation as a raw simple string literal. This
is true even when the file type indicator is invalid, as with `#"""foo"""#`.
```